### PR TITLE
restructure and updates in the keychain secion, moved the run a local…

### LINKED ISSUE #709: Keychain: review & restructure
Revisit it after we finalize the EVM update. Manually migrate the changes to a new branch.

**TO DO 1** - overall structure

- CLIChain: Install --> Run a Keychain from CLI -- done
- Revisit the titles and structure in the Keychain section -- done
- Edit the main Introduction to represent the new structure -- done
- Fix identation in some of the guides (4 spaces -> 3 spaces) -- done

**TO DO 2** - in-depth editing

- In Create a Keychain, mention Join Buenavista -- done
- Mention creating a Keychain in the Build a Keychain section -- done

### LINKED ISSUE #773: Run a local chain - move to Operate a node
**TO DO**

- Move the guide to Operate a node -- done
- Fix the links -- done
- Revisit the introduction of the section -- done

**BLOCKED: Do it only after we merge the Keychain and EVM updates (to avoid conflicts)**

### PR DESCRIPTION
**Pull request by ijonele**
- Implemented the new structure in the Keychain section
- Added two docs from the **docs-keychain** branch: CLIChain and WardenKMS
- Published the first part of Quick start as Create a Keychain
- Published the second part of Quick start as Set up a Basic Go app
- For now, we also have a more advanced guide that is not connected with the basic one. Should we keep it?
- In the **docs-keychain** branch, we have untested drafts of Set up a Basic Go app (a new version) and Build a keychain app (advanced), which are going to replace both guides.
- Moved all guides from the Test section

## Release Notes

- **New Features**
  - Introduced a comprehensive guide for creating and configuring a Keychain entity.
  - Added documentation for fulfilling key and signature requests using the CLIChain tool.

- **Documentation Updates**
  - Enhanced clarity and structure in multiple Keychain-related documents.
  - Added new sections, including commands for registering a Keychain, adding a Keychain Writer, and detailed instructions for using CLIChain, to streamline the process of building Keychain applications.

### CODE REVIEW

File: docs/developer-docs/docs/build-a-keychain/build-a-keychain-service/build-a-keychain-service.md
Comment (alijnmerchant21) [inline docs/developer-docs/docs/build-a-keychain/build-a-keychain-service/build-a-keychain-service.md:1]: Let us remove this file. I don't know if it works.

File: docs/developer-docs/docs/build-a-keychain/build-a-keychain-service/set-up-a-basic-go-app.md
Comment (alijnmerchant21) [inline docs/developer-docs/docs/build-a-keychain/build-a-keychain-service/set-up-a-basic-go-app.md:1]: Let us remove this as well.

**Review by alijnmerchant21 — COMMENTED** [review]
I'm rethinking the app example we have. IMO it should be a much more simple and stripped down app that simply creates a new key and sign a tx

File: docs/developer-docs/docs/build-a-keychain/build-a-keychain-service/set-up-a-basic-go-app.md
Comment (ijonele) [reply docs/developer-docs/docs/build-a-keychain/build-a-keychain-service/set-up-a-basic-go-app.md:1]: Maybe we can keep it? It's part of the old quick start written by Antonio

**Review by alijnmerchant21 — APPROVED** [review]

**Pull request merged**